imports/api/processes/process.methods.js

```diff
@@ -61,35 +24,35 @@ Meteor.methods({
       throw new Meteor.Error('Not allowed!');
     }
 
-    check(formValues.title, String);
-    check(formValues.description, String);
-    check(formValues.readingMaterial, String);
-    check(formValues.capacity, Number);
+    // check(formValues.title, String);
+    // check(formValues.description, String);
+    // check(formValues.readingMaterial, String);
+    // check(formValues.capacity, Number);
 
     try {
-      const add = Processes.insert(
+      const add = await Processes.insert(
         {
           host,
           adminId: user._id,
           adminUsername: user.username,
+          title: formValues.title,
+          description: formValues.description,
+          readingMaterial: formValues.readingMaterial,
+          imageUrl,
+          capacity: formValues.capacity || 20,
           members: [
             {
               memberId: user._id,
               username: user.username,
-              profileImage: user.profileImage || null,
+              // profileImage: user.profileImage || "",
               joinDate: new Date(),
             },
           ],
-          documents: [],
-          meetings: [],
-          title: formValues.title,
-          description: formValues.description,
-          readingMaterial: formValues.readingMaterial,
-          capacity: formValues.capacity || 20,
-          imageUrl,
+          // documents: [],
```

Comment:
Reason these are commented?

Reply:
I think these would be good to have there, especially `meetings`, even empty at creation.

Reply:
yes but now with the schemas we can set defaultValue so for those I defiened the defaultValue empty array in schemas

Reply:
hmm default value as empty array? Cool!